apps/sms/js/thread_list_ui.js

```diff
@@ -515,7 +521,7 @@ var ThreadListUI = {
     };
 
     // Add delete option when list is not empty
-    if (ThreadListUI.noMessages.classList.contains('hide')) {
+    if (!ThreadListUI.container.classList.contains('hide')) {
```

Comment:
do we still need this condition ? I think showOptions is not actionnable when the container is empty, with your patch?